os_net_config/common.py

```diff
@@ -95,16 +95,17 @@ def get_noop():
 
 
 def log_exceptions(type, value, tb):
-    logger.exception(''.join(traceback.format_exception(
-        type, value, tb)))
+    logger.exception("".join(traceback.format_exception(type, value, tb)))
     # calls default excepthook
     sys.__excepthook__(type, value, tb)
 
 
 def configure_logger(log_file=False, verbose=False, debug=False):
-    LOG_FORMAT = ('%(asctime)s.%(msecs)03d %(levelname)s '
-                  '%(name)s.%(funcName)s %(message)s')
-    DATE_FORMAT = '%Y-%m-%d %H:%M:%S'
+    LOG_FORMAT = (
+        "%(asctime)s.%(msecs)03d %(levelname)s "
+        "%(name)s.%(funcName)s %(message)s"
+    )
+    DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
     logger = logging.getLogger("os_net_config")
     logger.handlers.clear()
     logger_level(logger, verbose, debug)
```

Comment:
This is probably a good idea to use the builtin logging configuration primitives:

https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig

Example use (probably a bit extreme, but it can serve as reference):

https://github.com/evecm/ecm/blob/ddab6714e31f7ca90b448ccaf421186071e84dca/ecm/admin/instance_template/settings.py#L261-L311